plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -40,6 +44,7 @@
     private boolean tableStatisticsEnabled = true;
     private boolean projectionPushdownEnabled = true;
     private Optional<String> hiveCatalogName = Optional.empty();
+    private int formatVersion = FORMAT_VERSION_SUPPORT_MIN;
```

Comment:
This is compatible, but please follow up to create v2 by default.
We are implementing v2 deletes, and are currently not working on copy-on-write delete mode (as required for v1), so v2 are better for end users.

Reply:
If `copy-on-write` isn't implemented, do make sure that `write.format.default` is `merge-on-read` then. Particularly for users who might interact with the table via other engines.

Reply:
Actually there are 3 configurations coming up with respect to `copy-on-write` vs `merge-on-read`:

https://github.com/apache/iceberg/blob/a2392355e2a9afe757b700409882f26737a5062e/docs/tables/configuration.md?plain=1#L75-L79

Reply:
Yep, at the moment neither merge-on-read or copy-on-write are supported so I didn't add those three options in here, but they are in the PR adding support for merge-on-read deletes: https://github.com/trinodb/trino/pull/11886/commits/8ebba99d0642db3b6b4fdddfa3aa449c369cd8be#diff-2af3e19a6b656640a7d0bb73114ef224953a2efa04e569b1fe4da953b2cc6d15R411